iree/compiler/Dialect/VM/IR/VMOps.cpp

```diff
@@ -744,11 +754,19 @@ static ParseResult parseRodataOp(OpAsmParser &parser, OperationState *result) {
       failed(parser.parseAttribute(valueAttr, "value", result->attributes))) {
     return failure();
   }
+
   return success();
 }
 
 static void printRodataOp(OpAsmPrinter &p, RodataOp &op) {
   p << op.getOperationName() << ' ';
+
+  // TODO(#4670): Share across ops or upstream a custom directive
```

Comment:
I see we dont print all the attributes on this op. The canonical way of doing this is to print the attribute list on the op, there is a stock method to do that which excepts a list of already printed attribues. Anyway, just an FYI. Its not needed here.

Reply:
This custom printer/parser already skips some of the default formatting with its attributes. This special handling for symbol visibility as a bare word op prefix is forked from upstream:
* [visibility parsing](https://github.com/llvm/llvm-project/blob/16b284e99e4bca8d1e571310f545f10512dc979b/mlir/lib/IR/FunctionImplementation.cpp#L197-L198)
* [visibility printing](https://github.com/llvm/llvm-project/blob/16b284e99e4bca8d1e571310f545f10512dc979b/mlir/lib/IR/FunctionImplementation.cpp#L355-L357)